src/news/johnny-five-1-point-0.md

```diff
@@ -0,0 +1,18 @@
+---
+author: Bryan Hughes and Lyza Danger Garder
+date: '2018-08-23 00:00:00'
+status: published
+title: 'Johnny-Five v1.0'
+slug: 'v1_0'
+category:
+  - Announcement
+  - Platform
+---
+
+Hi from [JSConf US](https://2018.jsconf.us/) in sunny San Diego, Calif., where a representative group of Johnny-Five contributors [met and planned](https://github.com/rwaldron/johnny-five/issues/1463) what's next for Johnny-Five. And—ta-da!—we are thrilled to announce the release of Johnny-Five v1.0!
```

Comment:
Should we link `Johnny-Five v1.0` to the (to-be-released) link on `npm`?

Reply:
I kinda think we should list it as `1.0.0` instead of `1.0` to help drive home that this is about semver. What do you think?

Reply:
Agreed on ```1.0.0```